packages/contracts-bedrock/test/Safe/LivenessModule.t.sol

```diff
@@ -126,30 +153,36 @@ contract LivenessModule_CanRemove_Test is LivenessModule_TestInit {
     }
 }
 
-contract LivenessModule_Get75PercentThreshold_Test is LivenessModule_TestInit {
-    /// @dev check the return values of the get75PercentThreshold function against manually
-    ///      calculated values.
-    function test_get75PercentThreshold_Works() external {
-        assertEq(livenessModule.get75PercentThreshold(20), 15);
-        assertEq(livenessModule.get75PercentThreshold(19), 15);
-        assertEq(livenessModule.get75PercentThreshold(18), 14);
-        assertEq(livenessModule.get75PercentThreshold(17), 13);
-        assertEq(livenessModule.get75PercentThreshold(16), 12);
-        assertEq(livenessModule.get75PercentThreshold(15), 12);
-        assertEq(livenessModule.get75PercentThreshold(14), 11);
-        assertEq(livenessModule.get75PercentThreshold(13), 10);
-        assertEq(livenessModule.get75PercentThreshold(12), 9);
-        assertEq(livenessModule.get75PercentThreshold(11), 9);
-        assertEq(livenessModule.get75PercentThreshold(10), 8);
-        assertEq(livenessModule.get75PercentThreshold(9), 7);
-        assertEq(livenessModule.get75PercentThreshold(8), 6);
-        assertEq(livenessModule.get75PercentThreshold(7), 6);
-        assertEq(livenessModule.get75PercentThreshold(6), 5);
-        assertEq(livenessModule.get75PercentThreshold(5), 4);
-        assertEq(livenessModule.get75PercentThreshold(4), 3);
-        assertEq(livenessModule.get75PercentThreshold(3), 3);
-        assertEq(livenessModule.get75PercentThreshold(2), 2);
-        assertEq(livenessModule.get75PercentThreshold(1), 1);
+contract LivenessModule_GetRequiredThreshold_Test is LivenessModule_TestInit {
+    /// @dev Tests if getRequiredThreshold work correctly by implementing the same logic in a different manner
+    function _getLeastIntegerValueAbovePercentage(
+        uint256 _total,
+        uint256 _percentage
+    )
+        internal
+        pure
+        returns (uint256)
+    {
+        require(_percentage > 0 && _percentage <= 100);
+        uint256 toAdd;
+
+        // If the total multiplied by the percentage is not divisible by 100, we need to add 1 to the result to
+        // compensate for the rounding down by integer division.
+        if ((_total * _percentage) % 100 > 0) {
+            toAdd = 1;
+        }
+        return (_total * _percentage) / 100 + toAdd;
+    }
+
+    /// @dev Differentially tests the getRequiredThreshold function against _getLeastIntegerValueAbovePercentage
+    function testFuzz_getRequiredThreshold_works(uint256 _numOwners, uint256 _percentage) external {
```

Comment:
I liked the assertions for specific known values, especially if we can use values that would be reasonable for the SC safe to have. I'm pretty confident this code is correct and the fuzz is good but a few tests for known-good values would be safer.